### PR TITLE
Several escaped editors

### PR DESCRIPTION
**Pull request by ivanmalagon**
Sanitized some Builder inputs.

### CODE REVIEW

**Comment by nobuti** [comment]
That could not be right... No issue linked, that branch's name...

File: lib/assets/core/javascripts/cartodb3/components/inline-editor/inline-editor-view.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var CoreView = require('backbone/core-view');
 var _ = require('underscore');
+var cartodb = require('cartodb.js');
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/components/inline-editor/inline-editor-view.js]: I saw this as `var cdb = require('cartodb.js');`, I think we should keep tied to this convention.

**Comment by nobuti** [comment]
I found an issue:

alert

File: lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ var CoreView = require('backbone/core-view');
 var templateForm = require('./form.tpl');
 require('../../../form-components/index');
 var TipsyTooltipView = require('../../../tipsy-tooltip-view');
+var utils = require('../../../../helpers/utils');
```
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js]: Why isn't it  `Utils`? with capital letter.

File: lib/assets/core/test/spec/cartodb3/components/inline-editor/inline-editor-view.spec.js [hunk 3]
```diff
@@ -69,6 +69,23 @@ describe('components/inline-editor/inline-editor-view', function () {
     view2.remove();
   });
 
+  it('.getValue', function () {
+    view = new InlineEditorView({
+      template: template,
+      onEdit: onEdit,
+      onClick: onClick,
+      renderOptions: {
+        name: '<img src="http://emojipedia-us.s3.amazonaws.com/cache/b8/b4/b8b4e86a110557e6b6d666c9cf6d6cc8.png" />'
```
Comment (jorgesancha) [inline lib/assets/core/test/spec/cartodb3/components/inline-editor/inline-editor-view.spec.js]: Nice ;-)

File: lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js (hunk 2 above)
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js]: Ping!

Comment (ivanmalagon) [inline lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js]: It's not a constructor but an instance of an object with several properties. Hence, no constructor, no capital.

Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/modals/dataset-metadata/form/form-view.js]: Bueeeeno

**Comment by ivanmalagon** [comment]
Deployed to Ded02, ready for some kind soul to pass the acceptance.

**Comment by nobuti** [comment]
It works ✨

**Comment by nobuti** [comment]
🚀

**Pull request merged**